service/routes.py

```diff
@@ -5,8 +5,8 @@
 """
 
 from flask import jsonify, request, url_for, abort
-from .models import Inventory
 from .common import status  # HTTP Status Codes
+from service.models import Inventory
```

Comment:
Why did we need to change this?

Reply:
The service from service.models was removed for ensuring build was successful in terms of linting. But since we don't need to ensure that linting is required for build added service back as wanted to ensure the code works correctly.